MaxText/pyconfig.py

```diff
@@ -145,18 +145,26 @@ def update_model_vars(raw_keys):
     ''' Update model config variables
     '''
     validate_model_name(raw_keys['model_name'])
-    if raw_keys['model_name'] == 'llama-7b':
+    if raw_keys['model_name'] == 'llama2-7b':
       max_logging.log(f"Running Model: {raw_keys['model_name']}")
-      llama_7b_model_vars = {
+      llama2_7b_model_vars = {
         'base_emb_dim': 4096,
-        'num_query_heads': 32,
-        'num_kv_heads': 32,
+        'base_num_query_heads': 32,
+        'base_num_kv_heads': 32,
         'base_mlp_dim': 11008,
         'base_num_decoder_layers': 32,
         'head_dim': 128,
-        'mlp_activations': ['silu'],
+        'mlp_activations': ['silu','linear'],
+        'vocab_size': 32000,
+        'enable_dropout': False,
+        'attention':'dot_product',
+        'vocab_relative_path':'tokenizer.llama2',
```

Comment:
where does this tokenizer come from? (Might need to be in assets or something?)

Reply:
yes it is put in a GCS bucket

Reply:
should we put a comment saying explaining this?

Reply:
But a random GCS bucket? Should we jsut put it into MaxText's assets?

Reply:
As discussed offline, we are skipping this.